index.bs

```diff
@@ -2694,34 +4558,177 @@ partial interface MLGraphBuilder {
         - *SumSquare*: Compute the sum of the square of all the input values along the axes.
 </div>
 
-### The relu() method ### {#api-mlgraphbuilder-relu}
+<details open algorithm>
+
+  <summary>
+    To <dfn for="MLGraphBuilder" data-lt="reduce-op">create reduce operation</dfn> given |op|, |input| and |options|, run the following steps:
+  </summary>
+  <div class=algorithm-steps>
+    1. [=Assert=]: |op| is one of "reduceL1", "reduceL2", "reduceLogSum", "reduceLogSumExp", "reduceMax", "reduceMean", "reduceMin", "reduceProduct", "reduceSum", "reduceSumSquare".
+    1. [=Assert=]: the type of |input| is {{MLOperand}}.
+    1. If |options|.{{MLReduceOptions/axes}} [=map/exists=], if any of its elements is not in [=the range=] 0 to the [=rank=] of |input|, exclusive, then [=exception/throw=] a "{{DataError}}" {{DOMException}}.
```

Comment:
in [=the range=] 0 to  N-1 where N is the [=rank=] of the input tensor.

Reply:
The "exclusive" word does that. :)